test/source/tests/setup.ts

```diff
@@ -2479,9 +2479,56 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
         );
       })
     );
+
+    test(
+      'setup - check custom authentication config from the local store (customer url fes)',
+      testWithBrowser(async (t, browser) => {
+        t.context.mockApi!.configProvider = new ConfigurationProvider({
+          fes: {
+            authenticationConfiguration: {
+              oauth: {
+                redirectUrl: 'https://example.com/redirect',
+                clientId: 'your-client-id',
+                clientSecret: 'your-client-secret',
+                authCodeUrl: 'https://example.com/auth',
+                tokensUrl: 'https://example.com/tokens',
+              },
+            },
+          },
+        });
+        const acctEmail = 'user@authentication-config-test.flowcrypt.test';
+        await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+        const settingsPage = await browser.newExtensionSettingsPage(t, acctEmail);
+        const debugFrame = await SettingsPageRecipe.awaitNewPageFrame(settingsPage, '@action-show-local-store-contents', ['debug_api.htm']);
+        await debugFrame.waitForContent('@container-pre', 'authentication');
+        const key = `cryptup_${emailKeyIndex(acctEmail, 'authentication')}`;
+        const auth = (await settingsPage.getFromLocalStorage([key]))[key];
+        const { oauth } = auth as FesAuthenticationConfiguration;
+        expect(oauth.clientId).to.be.not.empty;
+        expect(oauth.clientSecret).to.be.not.empty;
+        expect(oauth.redirectUrl).to.be.not.empty;
+        expect(oauth.authCodeUrl).to.be.not.empty;
+        expect(oauth.tokensUrl).to.be.not.empty;
```

Comment:
I think it'll be better to compare these properties with real values, not just check if they're not empty. We can even move oauth config to separate variable and then compare with it:
```ts
const oauthConfig = {
  redirectUrl: 'https://example.com/redirect',
  clientId: 'your-client-id',
  clientSecret: 'your-client-secret',
  authCodeUrl: 'https://example.com/auth',
  tokensUrl: 'https://example.com/tokens',
};
t.context.mockApi!.configProvider = new ConfigurationProvider({
           fes: {
             authenticationConfiguration: {
               oauth: oauthConfig
             },
           },
         });
...
const { oauth } = auth as FesAuthenticationConfiguration;
expect(oauth).to.eq(oauthConfig);
```

Reply:
thank you very much - the one you have suggested is by far the best way to check JSON values instead of the way I tried to test them. Thanks again - I learned a lot.

Reply:
You're welcome :)

Reply:
Looks like `to.eq` doesn't work for objects, just for simple values. Let's try to use `to.deep.equal`

Reply:
Thanks - `to.deep.equal` seems to be used across the codebase and worked for checking objects.